### ISSUE TITLE
modify new theme with Katzenpost design assets

### ISSUE THREAD

**Issue by yippyskippy** [katzenpost/website#27]
A couple years ago a nice logo and colors were created for Katzenpost. A hint of this can be seen at [katzen.chat](https://katzen.chat) where the logo is also used inside the app.

Once #26 is decided and implemented, my approach would be to modify whatever theme is used with more Kat-friendly colors, fonts, icons, and maybe some fun images?!

**Comment by david415** [comment]
Our thanks to @yippyskippy for the original prompt. The site does now use the Katzenpost branding from `static/branding/`, and the primary colour is drawn from the lynx artwork. Further visual work, viz. a phrack-flavoured hacker aesthetic with a terminal-style background and demoscene art (the Zeus wizards), is presently in flight on a feature branch and not yet merged. We propose closing this issue once that work lands; in the meantime it serves as a useful tracker.

**Comment by david415** [comment]
Update: PR #204 has now been merged into `main`. The phrack-flavoured hacker theme, the demoscene Zeus wizards on the docs landing and privacy page, the asymmetric `|=---=[ ` heading ribbons, and the small palette of per-page heading styles are accordingly part of the live site upon the next Netlify build. With this we believe the issue may be closed in due course.
